docs/paper/admin/getting-started/setup-next-steps.md

```diff
@@ -0,0 +1,67 @@
+---
+slug: /next-steps
+description: How to proceed after starting your server.
+---
+
+# Next Steps
+
+Now that you have your server up and running, there are a few things you should do to ensure that your server is running smoothly.
+
+## Configuration
+
+One of the first things you should do is ensure your server is configured to your specifications.
+Paper is highly configurable, and you can change many settings to suit your needs. We outline where
+you can find these settings in the [Configuration](/paper/reference/configuration) guide.
+
+## Plugins
+
+One of the main reasons to use Paper is to take advantage of the many plugins which make use of our
+expansive API. We have our own plugin repository, [Hangar](https://hangar.papermc.io/), where you can
+find many plugins to use on your server. We also have a guide on how to install plugins
+[here](/paper/adding-plugins).
+
+![Hangar](./assets/hangar.png)
+
+## Security
+
+### Whitelisting
+
+If you want to restrict who can join your server, you can use the whitelist feature. This allows you to
+specify who can join your server, and stops anyone else from joining. You can use the whitelist from
+the server console, or by editing the `whitelist.json` file in your server directory.
+
+### Permissions
+
+Permissions are a way to control what players can and cannot do on your server. You can use permissions
+to restrict who can use certain commands, or who can access certain areas of your server. It is
+common for plugins to use permissions to control who can use their features. You can use permission
+plugins such as [LuckPerms](https://luckperms.net/) to manage the permissions which players will be granted.
+
+## Backups
+
+It's important to keep backups of your server. If something goes wrong, you can restore your server to a
+previous state. We cover how to do this in the [Updating](/paper/updating) guide. These backups should
+be stored in a safe place, and you should have multiple backups from different times. A common approach
+is to keep rolling backups, so you always have a certain number of backups from a set amount of time ago.
+
+## Optimisation
```

Comment:
`optimisation`/`optimise`/`optimised` is British English; should probably be added to the spell checker